index.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap');
+      </style>
```

Comment:
- [ ] [Optional] 👍 Great job on your project! I noticed that you have included custom font styles using internal CSS in your HTML file. However, it would be better to utilize the existing style.css file for importing the fonts.

📝 By importing the fonts in the style.css file, you can keep your CSS code more organized and separate concerns between the HTML and CSS files. This will also allow you to easily manage and update the fonts in one central location.

🚀 Please consider making this change in your project.💡 To do this, you can add an @import rule at the top of your style.css file to import the desired font files.